third_party/expect/src/lib.rs

```diff
@@ -6,7 +6,7 @@ macro_rules! expect_eq {
         match (&$left, &$right) {
             (left_val, right_val) => {
                 if !(*left_val == *right_val) {
-                    return Err(std::io::Error::new(
+                    return Err(Box::new(std::io::Error::new(
```

Comment:
I think you could avoid this manual box too, and let `?` handle it.

Reply:
Doesn't seem to work &ndash; there's a bunch of "expected struct `std::boxed::Box`, found struct `std::io::Error`" (maybe because this is a macro not a function call?)